teamspeak/stack.yml

```diff
@@ -1,24 +1,24 @@
 version: '3.1'
 services:
-    teamspeak:
-        image: teamspeak-server
-        restart: always
-        ports:
-          - 9987:9987/udp
-          - 10011:10011
-          - 30033:30033
-        environment:
-            TS3SERVER_DB_PLUGIN: ts3db_mariadb
-            TS3SERVER_DB_SQLCREATEPATH: create_mariadb
-            TS3SERVER_DB_HOST: db
-            TS3SERVER_DB_USER: root
-            TS3SERVER_DB_PASSWORD: example
-            TS3SERVER_DB_NAME: teamspeak
-            TS3SERVER_DB_WAITUNTILREADY: 30
-            TS3SERVER_LICENSE: accept
-    db:
-        image: mariadb
-        restart: always
-        environment:
-            MYSQL_ROOT_PASSWORD: example
-            MYSQL_DATABASE: teamspeak
+  teamspeak:
+    image: teamspeak-server
```

Comment:
Whoops, missed one!  This should be just `teamspeak`, right? (I can fix this post-merge if we merge before you get to it)